src/Umbraco.Web.UI.Client/src/views/propertyeditors/listview/listview.controller.js

```diff
@@ -304,6 +304,14 @@ function listViewController($scope, $interpolate, $routeParams, $injector, $time
                     navigationService.reloadSection(section);
                 }
             }
+        }, function(error){
```

Comment:
Nitpicking a bit, but could we use `.catch(function (error) {})` instead of using the second argument of `.then`?

Reply:
Aye 👍 fixed in latest.